src/parserstate.c

```diff
@@ -355,7 +355,7 @@ parserstate *alloc_parser(rbs_string_t string, const rbs_encoding_t *encoding, i
   return parser;
 }
 
-void rbs_parser_declare_type_variables(parserstate *parser, size_t count, const char *variables[count]) {
```

Comment:
TIL MSVC supports C11 and C17, [but not C99 VLAs](https://devblogs.microsoft.com/cppblog/c11-and-c17-standard-support-arriving-in-msvc/#what’s-not:)